README.md

```diff
@@ -87,7 +87,11 @@ Getting started
    are not tracked by Git (optional):
 
     ```
-    coverage
+    echo "coverage" >> .gitignore
+    ```
+    Or if you use Windows:
+    ```
+    echo coverage >> .gitignore
```

Comment:
am I missing something or is the only difference the quotes? If so, you can omit quotes on *nix as well. 

My own feeling is that if you can't be trusted to know how to add a line to your gitignore you have bigger issues.